WebInterface/src/components/PrivacyAndTerms.tsx

```diff
@@ -0,0 +1,197 @@
+import React from "react";
+
+export const PrivacyAndTerms = () => {
+    return (
+        <div className="max-w-7xl mx-auto p-6">
+            <h1 className="text-2xl font-bold mb-4">Privacy and Terms</h1>
+
+            <section className="mb-8">
+                <h2 className="text-xl font-semibold mb-4">Datenschutzrichtlinie (Privacy Policy)</h2>
```

Comment:
German is really cool but I'm not translating this myself to check if it actually means something. Why no English at all? Shouldn't be hard to vibetranslate